UnitTests/UnitTests.Notifications.UWP/UnitTests.Notifications.UWP.csproj

```diff
@@ -110,6 +149,9 @@
     <PackageReference Include="MSTest.TestFramework">
       <Version>1.2.0</Version>
     </PackageReference>
+    <PackageReference Include="System.Net.NameResolution">
```

Comment:
Side effect?  Did you update the notifications unit tests?

Reply:
This change should have been rolled back by the undo - I was poking at the Unit Tests to see if I could get unit tests to stop running automatically.